tests/k8s_testcases/example_policies/online_boutique/orig_online_boutique_manifests/istio-manifests.yaml

```diff
@@ -48,39 +48,6 @@ spec:
           number: 80
 ---
 apiVersion: networking.istio.io/v1alpha3
-kind: ServiceEntry
```

Comment:
the tests using this file, ignored ServiceEntry objects in their results (it was not supported - so acted as if these objects are not existed)
my changes - would change the topology and results of those tests; so I chose to remove the  `ServiceEntry` objects in order to keep their results as is

Reply:
I think it might be interesting to analyze with these resources as well, so maybe try to create another instance of this test containing them as well, and check how the expected output changes

Reply:
sure

Reply:
reverted this change (returning the ServiceEntry objects), after committing last fixes, the existence of ServiceEntry objects in this test will not affect the result since it contains only k8s policies